### DISCUSSION TITLE
Toast: Add severity to icon slotProps

### DISCUSSION THREAD

**Discussion by MarcelGeo** [primefaces/community#928] [PrimeVue]
In documentation to deprecated prop ```warnIcon``` is defined icon slot as replacement to this. It could be useful to have severity or whole message object in slotProps of #icon.

Expected behavior
```vue
<Toast>
  <template #icon="{ severity }">
    {{ severity based icon }}
  </template>
</Toast>
```

**Comment by tugcekucukoglu** [comment]
We can wait for community demand.
